Gruntfile.js

```diff
@@ -80,6 +81,15 @@ module.exports = function(grunt) {
         jshintrc: true
       },
       main: allFiles
+    },
+    watch: {
+      src: {
+        files: ['src/*', 'pointerevents.js'],
+        tasks: ['build', 'lint'],
```

Comment:
I prefer watch tasks to build as fast as possible, so that any reload in a browser will get up-to-date files, even when linting fails.